### PR TITLE
Fixing #issue139

### PR DESCRIPTION
**Pull request by felipevolpone**
I'm doing the #139 improvement. I think that I made in the right place, but please tell me if I did something wrong.

### CODE REVIEW

**Comment by felipevolpone** [comment]
@gwelymernans I think that is right now.

File: yapftests/yapf_test.py [hunk 1]
```diff
@@ -573,9 +573,9 @@ def g():
 
 
 
-        #comment
```
Comment (hayd) [inline yapftests/yapf_test.py]: I thought the point of these not changing is that they are outside of the range 3 3.

Presumably this means that your change is not checking for being within the range(s).

**Comment by bhack** [comment]
Any news on this?

**Comment by govjess** [comment]
The hardest part is channeling the money where it doesn't get hacked in the process by anyone who can socialize with me online. Making a paper billfold is mandatory and not to give keys online at anytime to the private wallet.

**Comment by gwelymernans** [comment]
I think the fix this PR is meant to handle already exists now.

**Pull request closed without merging**